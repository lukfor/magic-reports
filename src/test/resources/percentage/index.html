{{percentage(double)}}
{{percentage(float)}}
{{percentage(string)}}
{{percentage("1")}}