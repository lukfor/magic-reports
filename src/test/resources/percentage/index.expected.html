51.11%
71.21%
NaN%
NaN%